docs/config/README.md

```diff
@@ -2642,6 +2642,7 @@ The module will be shown if any of the following conditions are met:
 - The current directory contains a `Gemfile` file
 - The current directory contains a `.ruby-version` file
 - The current directory contains a `.rb` file
+- The environment variables `RUBY_VERSION` or `RBENV_VERSION` are set
```

Comment:
Should we mention that the value of these env vars will not be used? If the `RUBY_VERSION` is set to x but `ruby - - version` outputs y this might cause some confusion.

Reply:
Perhaps we could add a message below the bullet points like this:
"Starship gets the current Ruby version by checking `ruby -v`."
The same thing applies to the current directory files as well.